docs/admin/configuration.md

```diff
@@ -49,10 +49,12 @@ The following table shows the environment variables available for Perseo configu
 | PERSEO_AUTHENTICATION_USER         | User to perform authentication                                                                                                                                                                          |
 | PERSEO_AUTHENTICATION_PASSWORD     | Password for the user to perform authentication                                                                                                                                                         |
 | PERSEO_AUTHENTICATION_SERVICE      | Keystone User Service to perform authentication                                                                                                                                                         |
-| PERSEO_CAST_TYPE                   | If true, enable attribute value casting based in NGSI-v2 attribute types if true. If false (default), the JSON native type for the attribute value is used.                                             |
-
-| PERSEO_MAX_RULES_BY_CORR | Maximum number of rule executions triggered by the same notification (with the same correlator) until refuse execute a rule. Default is 20. See [loop detection tips](https://github.com/telefonicaid/perseo-fe/blob/master/docs/architecture/architecture.md#loop-detection)
-|
+| PERSEO_CAST_TYPE                   | Maximum number of rule executions triggered by the same notification (with the same correlator) until refuse execute a rule. Default is 20. See [loop detection tips](https://github.com/telefonicaid/perseo-fe/blob/master/docs/architecture/architecture.md#loop-detection)   |
+| PERSEO_MAX_RULES_BY_CORR           | If true, enable attribute value casting based in NGSI-v2 attribute types if true. If false (default), the JSON native type for the attribute value is used.                                           |
+| PERSEO_CHECK_NON_SIGNAL_BY_API     | If false, the entities will be directly find using MongoDB, if true they will be fing using NGSIv2 API*                 
+                                                
+'*' Although the idea is that now Perseo works always using NGSIv2 API to check times for non signal rules, this setting has been introduced to ease a potential rollback if some problem is found with this new approach. Thus, I has to be considered a kind of temporal debugging settings, to be removed when we get confident with the new functionality, thus you shouldn't use it with care. 
```

Comment:
```suggestion
| PERSEO_CHECK_NON_SIGNAL_BY_API     | If false, the entities will be directly find using MongoDB, if true they will be fing using NGSIv2 API (*)                 
                                                
(*) Although the idea is that now Perseo works always using NGSIv2 API to check times for non signal rules, this setting has been introduced to ease a potential rollback if some problem is found with this new approach. Thus, I has to be considered a kind of temporal debugging settings, to be removed when we get confident with the new functionality, thus you shouldn't use it with care. 
```

Reply:
solved in this commit:
https://github.com/telefonicaid/perseo-fe/pull/726/commits/b0ba12e4c9a979b733fe7acbd108923fa0e4c257